dumpling/dumpling_test.go

```diff
@@ -72,7 +77,11 @@ func (d *testDumplingSuite) SetUpSuite(c *C) {
 			Dir: dir,
 		},
 		BAList: &filter.Rules{
-			DoDBs: []string{"information_schema"},
+			DoDBs: []string{testDumplingSchemaName},
```

Comment:
what's the purpose of changing this test

Reply:
reason is here https://github.com/pingcap/dm/pull/1910#issuecomment-894554256